ground/gcs/gcs.pro

```diff
@@ -20,4 +20,9 @@ unix:!macx:!isEmpty(copydata) {
 	QMAKE_EXTRA_TARGETS += bin_install
 }
 
+!equals(QT_MAJOR_VERSION, 5) | !equals(QT_MINOR_VERSION, 6) | !equals(QT_PATCH_VERSION, 1) {
+message("Cannot build dRonin GCS with Qt version $${QT_VERSION}.")

```

Comment:
I'm not sure we even need to worry about the patch level, but I suppose it doesn't hurt (most of the time).